ci/docker/uc-dataapp_resources_provider/application.properties

```diff
@@ -0,0 +1,38 @@
+spring.banner.location=classpath:true-connector-uc-banner.txt
+
+springfox.documentation.open-api.v3.path=/api-docs
+server.servlet.contextPath=/platoontec/PlatoonDataUsage/1.0
+#
+#/platoon-data-usage
+#
+server.port=8180
```

Comment:
I think you mixed up the ports for UC app

uc-dataapp-provider:
    image: rdlabengpa/ids_uc_data_app_platoon:v1.3
    environment:
      - TZ=Europe/Rome
    expose:
      - "8280"